service/frontend/workflow_handler.go

```diff
@@ -2119,44 +2101,78 @@ func (wh *WorkflowHandler) ListOpenWorkflowExecutions(ctx context.Context, reque
 		return nil, err
 	}
 
-	baseReq := &manager.ListWorkflowExecutionsRequest{
-		NamespaceID:       namespaceID,
-		Namespace:         namespaceName,
-		PageSize:          int(request.GetMaximumPageSize()),
-		NextPageToken:     request.NextPageToken,
-		EarliestStartTime: request.StartTimeFilter.EarliestTime.AsTime(),
-		LatestStartTime:   request.StartTimeFilter.LatestTime.AsTime(),
+	if request.StartTimeFilter == nil {
+		request.StartTimeFilter = &filterpb.StartTimeFilter{}
+	}
+
+	earliestTime := request.StartTimeFilter.EarliestTime
+	latestTime := request.StartTimeFilter.LatestTime
```

Comment:
You need to use the Get version to avoid panic trying to access fields in a nil object: `GetEarliestTime` and `GetLatestTime`.